src/main/java/seedu/address/model/leave/Status.java

```diff
@@ -40,7 +40,7 @@ public static Status of(String status) throws IllegalArgumentException {
         if (!isValidStatus(status)) {
             throw new IllegalArgumentException(MESSAGE_CONSTRAINTS);
         }
-        return new Status(StatusType.valueOf(status));
+        return new Status(StatusType.valueOf(status.toUpperCase()));
```

Comment:
Are we allowed to change this? Does this change fall under fixing a bug or is it a feature implementation? I'm afraid that we might accidentally violate v1.4 rules.

Perhaps we should check with prof before implementing this.